lib/core/src/lib/common/utils/date-fns-utils.ts

```diff
@@ -78,4 +79,72 @@ export class DateFnsUtils {
         }
         return dateFnsLocale;
     }
+
+    /**
+     * A mapping of Moment.js format tokens to date-fns format tokens.
+     */
+    static momentToDateFnsMap = {
+        M: 'M',
```

Comment:
Do we require this M: 'M'? Actually, we are not transforming it in any method.

Reply:
removed